From fafe0abcdea360e9ecaab31048c7aa6003c97a6b Mon Sep 17 00:00:00 2001
From: Demon000 <demonsingur@gmail.com>
Date: Thu, 11 Jan 2018 08:51:21 +0200
Subject: [PATCH 06/20] legacy: add back nfc rules

Removed in b9b7eae7366c402e1c0777348b694f3bb859e2e7, might
still be necessary for old blobs.

Change-Id: Ie6508c882226ed7591916f05e559cf6421bada6b
---
 legacy-common/nfc.te | 3 +++
 1 file changed, 3 insertions(+)
 create mode 100644 legacy-common/nfc.te

diff --git a/legacy-common/nfc.te b/legacy-common/nfc.te
new file mode 100644
index 0000000..92af58c
--- /dev/null
+++ b/legacy-common/nfc.te
@@ -0,0 +1,3 @@
+qmux_socket(nfc)
+
+allow nfc self:socket create_socket_perms_no_ioctl;
-- 
2.7.4

